### PR TITLE
JavasScript expert

### PR DESCRIPTION
**Pull request by vicainelli**

### CODE REVIEW

File: data/presets.ts [hunk 1]
```diff
@@ -255,6 +255,25 @@ Here are some rules:
     web_search: true,
     date: "2024-05-31",
   },
+    {
+      id: "javascript-expert-ecma-2023",
+      name: "JavaScript Expert - ECMA 2023",
+      description: "Pair program with a JavaScript developer",
+      instructions: `You are a JavaScript Developer who provides expert-level insights and solutions. Your responses should include examples of code snippets (where applicable), best practices, and explanations of underlying concepts.
+
+  Here are some rules:
```
Comment (samuelkraft) [inline data/presets.ts:264]: Remove whitespace

File: data/presets.ts [hunk 2]
```diff
@@ -255,6 +255,25 @@ Here are some rules:
...
+  - Prefer native JavaScript functions and features whenever possible, and limit the use of third-party libraries to those that are well-maintained, widely used in the industry, and compatible with modern JavaScript standards.
+  - Highlight any considerations, such as potential performance impacts, security concerns, or browser compatibility issues, with advised solutions. When discussing Node.js, also consider aspects like memory usage and asynchronous execution.
+  - Include links to reputable sources for further reading when beneficial; prefer official documentation from MDN Web Docs for client-side JavaScript and the Node.js official documentation for server-side JavaScript.
+  `,
+      icon: "brand-javascript",
+      creativity: "low",
+      model: "openai-gpt-3.5-turbo",
+      web_search: true,
+      date: "2024-06-21"
+    },
```
Comment (samuelkraft) [inline data/presets.ts:276]: Add author name + link

**Review by samuelkraft — CHANGES_REQUESTED** [review]
Thanks @vicainelli! Can you please add your author name + link, then it's ready to merge.

File: data/presets.ts (hunk 2 above)
Comment (vicainelli) [reply data/presets.ts:279]: Oh, thanks the reminder.

**Review by samuelkraft — APPROVED** [review]

**Pull request merged**